model/os/registry.yaml

```diff
@@ -53,6 +53,10 @@ groups:
               stability: development
             - id: z_os
               value: 'z_os'
+              brief: "Deprecated. Use `zos` instead."
```

Comment:
I think this needs to be moved to a `registry_deprecated.yaml` file. Like https://github.com/open-telemetry/semantic-conventions/blob/main/model/http/deprecated/registry-deprecated.yaml

Reply:
it's the enum member definition, we can't move a single member to a separate file